module/VuFind/src/VuFind/ILS/Driver/DAIA.php

```diff
@@ -35,7 +35,7 @@
 namespace VuFind\ILS\Driver;
 
 use DOMDocument;
-use Laminas\Log\LoggerAwareInterface as LoggerAwareInterface;
+use Psr\Log\LoggerAwareInterface as LoggerAwareInterface;
```

Comment:
Here's another import that can be simplified.

```suggestion
use Psr\Log\LoggerAwareInterface;
```